src/main/java/com/fcgl/madrid/forum/controller/PostController.java

```diff
@@ -46,4 +48,10 @@ public Health checkHealth() {
     public ResponseEntity<InternalStatus> post(PostRequest postRequest) {
         return postService.post(postRequest);
     }
+
+    @GetMapping(path="/post")
```

Comment:
shouldnt the path be getPost? will cause less confusion so that there isnt 2 post methods

Reply:
I agree :). Will change. Never really liked the convention that is generally used

Reply:
Done: https://github.com/fcgl/forum/pull/9/commits/72ff580ad9ff028694a24ad5ba513cd39c978332

switched `post` to `getPost`